corehq/apps/style/templates/style/base.html

```diff
@@ -276,7 +276,7 @@
                                     </div>
                                 {% endfor %}
                             {% endif %}
-                            <div id="message-alerts" data-bind="foreach: alerts">
+                            <div id="message-alerts" class="ko-template" data-bind="foreach: alerts">
```

Comment:
Yeah, I wanted it to be more explicit for where I call `this.removeClass('ko-template');` above. If there's some other trickery someone's trying to do with `hide`, I don't want that to get removed. The class `ko-template` is supposed to mean "this is conceptually just a chunk of code that shouldn't actually be in the DOM until it's rendered, but I'm storing it in the DOM because that's more convenient and efficient performance-wise", whereas "hide" is more imperative than semantic. Anyway, that was my thought process. What do you think?